### ISSUE TITLE
Question: All images are same with different DTB?

### ISSUE THREAD

**Issue by devikcode** [AeolusUX/ArkOS-R3XS#80]
What I did was:

- Use P4 image with win32 disk Manager
- Before use in my console, merge the original screen files in BIOS (because at this point I tought I have original screen)

It works.

So my question is: In this repo you have to choose 1 option to download, but all of them are the same with just the DTB files for this version?
Just to know it, because if there are different things, maybe I would re-do all the process with the original screen Arkos version

Thanks in advance, great job guys!

**Comment by AeolusUX** [comment]
Yes it's for people that know which panel they have so that they don't need to replace dtb.
